js/main.js

```diff
@@ -0,0 +1,46 @@
+const RendererMod = Renderer()
```

Comment:
your instance should be lowercase in JS
`const renderer = renderer()`
`const productManager = ProductManager()`

try not to rename the instances something different than your module